apps/studio/components/layouts/OrganizationLayout.tsx

```diff
@@ -32,30 +73,15 @@ const OrganizationLayout = ({ children }: PropsWithChildren<{}>) => {
           <ScaffoldTitle>{selectedOrganization?.name ?? 'Organization'} settings</ScaffoldTitle>
         </ScaffoldContainer>
         <ScaffoldContainer>
-          <nav>
-            <Tabs
-              listClassNames="border-none"
-              size="small"
-              type="underlined"
-              activeId={id}
-              onChange={(id: any) => {
-                router.push(`/org/${slug}/${id}`)
-              }}
-            >
-              <Tabs.Panel id="general" label="General" className="!my-0" />
-              <Tabs.Panel id="team" label="Team" className="!my-0" />
-
-              <Tabs.Panel id="integrations" label="Integrations" className="!my-0" />
-
-              <Tabs.Panel id="billing" label="Billing" className="!my-0" />
-              <Tabs.Panel id="usage" label="Usage" className="!my-0" />
-              {invoicesEnabled && <Tabs.Panel id="invoices" label="Invoices" className="!my-0" />}
-              <Tabs.Panel id="apps" label="OAuth Apps" className="!my-0" />
-              <Tabs.Panel id="audit" label="Audit Logs" className="!my-0" />
-
-              <Tabs.Panel id="documents" label="Legal Documents" className="!my-0" />
-            </Tabs>
-          </nav>
+          <NavMenu className="border-none" aria-label="Organization menu navigation">
+            {navMenuItems.map((item) =>
```

Comment:
What about filtering hidden items out instead of doing the ternary operator with `null`

Reply:
yup looks cleaner now, thanks!